_includes/sidebar.html

```diff
@@ -1,12 +1,21 @@
 <ul id="mysidebar" class="nav">
 
-  <li class="sidebar-title">Get started</li>
+  <li class="sidebar-title">Get Flutter SDK</li>
 
     <ul class="sidebar-items">
       <li><a href="/setup/">Install</a></li>
-      <li><a href="/getting-started/">Create and run your first app</a></li>
       <li><a href="/ide-setup/">Setting up the IDE</a></li>
-      <li><a href="/bootstrap-into-dart/">Bootstrap into Dart</a></li>
+    </ul>
+  
+  <li class="sidebar-title">Get started with Flutter</li>
+
+    <ul class="sidebar-items">
+      <li><a href="/getting-started/">Create and run your first app</a></li>
+      <li><a href="/learn/build/">Rendering UI with build()</a></li> 
```

Comment:
Do we mention "build()" here? That's an implementation detail.

I like the idea of the titles that are task oriented. "Rendering UI", "Managing state", etc...